src/tests/dictionaryConcepts/actions/dictionaryConcept.test.js

```diff
@@ -317,48 +316,6 @@ describe('Test suite for dictionary concept actions', () => {
     });
   });
 
-  it('should fetch dictionary concepts by name', () => {
-    moxios.wait(() => {
-      const request = moxios.requests.mostRecent();
```

Comment:
Are these tests removed because they were not correct?

Reply:
No, they are removed because I deleted some duplicate code. Particularly [here](https://github.com/openmrs/openmrs-ocl-client/pull/497/files#diff-d932a1b328dd7277dbdcffb6b4d1a561).